src/Admin/Editor.php

```diff
@@ -50,6 +57,12 @@ public function validate_before_save_cb( $data, $post ) {
 		return $data;
 	}
 
+	/**
+	 * Initialize Admin functionality for WPGraphQL
```

Comment:
This comment doesn't seem to describe this function. 

Isn't `is_valid_form` validating data rather than initializing admin functionality?

Reply:
Oh gosh.  :copy: 🍝